spaces/S000178/properties/P000048.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000178
+property: P000048
+value: true
+---
+
+If $x, y$ are two distinct points of $X$, then either one of them is of the form $a_{\alpha_0\beta_0}$ so the clopen set $U = \{a_{\alpha_0\beta_0}\}$ separates them, or none of them is of the form $a_{\alpha_0\beta_0}$, but one is of the form $c_{\gamma_0}$ so that the clopen set $U = \{c_{\gamma_0}\}\cup\{a_{\gamma_0\beta}:\beta < \omega_1\}$ separates them.
```

Comment:
```suggestion
If $x, y$ are two distinct points of $X$, then either one of them is of the form $a_{\alpha\beta}$ so the clopen set $U = \{a_{\alpha\beta}\}$ separates them, or none of them is of the form $a_{\alpha\beta}$, but one is of the form $c_{\gamma}$ so that the clopen set $U = \{c_{\gamma}\}\cup\{a_{\gamma\beta}:\beta < \omega_1\}$ separates them.
```